app/views/candidate_interface/submitted_application_form/submit_success.html.erb

```diff
@@ -22,9 +22,11 @@
     <p class="govuk-body">You can track or withdraw your application from your application dashboard.</p>
     <p class="govuk-body"><%= govuk_link_to 'To view your application, return to your application dashboard', candidate_interface_application_complete_path %>.</p>
 
-    <% unless FeatureFlag.active?('feedback_form') %>
-      <h2 class="govuk-heading-m">Tell us what you think of this service</h2>
-      <p class="govuk-body">Your feedback will help us improve.</p>
+    <h2 class="govuk-heading-m">Tell us what you think of this service</h2>
+    <p class="govuk-body">Your feedback will help us improve.</p>
+    <% if FeatureFlag.active?('feedback_form') %>
+      <%= govuk_button_link_to 'Give feedback', candidate_interface_feedback_form_path %>
+      <%else %>
       <%= govuk_button_link_to 'Give feedback', candidate_interface_satisfaction_survey_recommendation_path %>
```

Comment:
I think this can come out now. We shouldn't be using this survey at all going forward. I'll card up something to remove it entirely.

Reply:
Is this what you were thinking? Or should we remove the feature flag now too?

```    <% if FeatureFlag.active?('feedback_form') %>   
      <h2 class="govuk-heading-m">Tell us what you think of this service</h2>
      <p class="govuk-body">Your feedback will help us improve.</p>
      <%= govuk_button_link_to 'Give feedback', candidate_interface_feedback_form_path %>
    <% end %>
```

Reply:
I think we can remove the flag entirely. It's pretty old now and we know the form works as intended.